go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/TykTechnologies/tyk
 
-go 1.21.0
-
-toolchain go1.21.4
+go 1.22.2
```

Comment:
I think we can use `1.22.3`, it's used here as well https://github.com/TykTechnologies/tyk/pull/6230/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R8